src/emailservice/Dockerfile

```diff
@@ -1,32 +1,25 @@
-FROM python:2.7-alpine3.8 as base
+FROM python:3.7.0-alpine3.8 as base
```

Comment:
I recommend doing `python:3-alpine` here so that:

1. we get python 3.x patches
2. we get stable alpine patches (not _edge_)

Reply:
Will do.